gammapy/time/plot.py

```diff
@@ -30,3 +28,140 @@ def plot_time_difference_distribution(time, ax=None):
 
     # TODO: implement!
     raise NotImplementedError
+
+def plot_light_curve(name_3FGL, time_start, time_end, ax = None):
+    """Plot flux as a function of time for a fermi 3FGL object.
+
+    Parameters
+    ----------
+    name_3FGL : `string`
+        The 3FGL catalog name of the object to plot
+    fermi_met_start : `~astropy.time.Time`
+        Astropy time object for the start of the light curve
+    fermi_met_end : `~astropy.time.Time`
+        Astropy time object for the end of the light curve
+    ax : `~matplotlib.axes.Axes` or None
+        Axes
+    Returns
+    -------
+    ax : `~matplotlib.axes.Axes`
+
+    Examples
+    --------
+    Plot effective area vs. energy:
+
+    .. plot::
+        :include-source:
+
+        import gammapy.time
+        import astropy.time
+        import matplotlib.pyplot as plt
+
+        time_start = astropy.time.Time('2010-01-01T00:00:00')
+        time_end = astropy.time.Time('2015-02-02T02:02:02')
+
+        plt.plot = gammapy.time.plot_light_curve('3FGL J0349.9-2102', time_start, time_end)
+        plt.show()
+    """
+    from ..datasets import fetch_fermi_catalog
+    import astropy.time
+    import numpy as np
+    import matplotlib.pyplot as plt
+    import matplotlib.dates as mdates
+    import math
+
+    ax = plt.gca() if ax is None else ax
+
+    fermi_met_base = astropy.time.Time('2001-01-01T00:00:00')
```

Comment:
This reference time should not be hard-coded here, it will be needed in several places.
Note that MET time handling is one of the tasks that needs to be figure out / improved: #284

For now I think simply moving this to
https://github.com/gammapy/gammapy/blob/master/gammapy/time/utils.py
as a module-level constant and leaving a TODO marker would be the best short-term solution:
```python
# TODO: implement and document this properly.
# see https://github.com/gammapy/gammapy/issues/284
TIME_REF_FERMI = Time('2001-01-01T00:00:00')
```